index.html

```diff
@@ -56,6 +56,17 @@
     </div>
   </nav>
 
+  <!-- Header -->
+  <header id="header-index">
+    <div class="header-text">
+      <h1>Dental Clinic</h1>
+      <p>3221 Waialae Ave Ste 320
+        <br>Honolulu HI 96816
+      </p>
+    </div>
+    <img src="images/dentalheaderimage.svg" alt="" id="header-img">
```

Comment:
Hey Mel, be sure to add something to your alt attribute. This is important for Accessibility purposes, as well as improving our clients SEO ranking on search engine results pages. Keep up the great work!

Reply:
Consider it done :) Thanks for the insightful review!